### PR TITLE
Bug/chapter1 vsts references

### PR DESCRIPTION
**Pull request by rtadd-git**
Updates to chapter 1 to remove references to VSTS. Whilst I was at it I also updated references to how widely git's used and tweaked a couple of proof reading things.

**Review questions:**
1. Have I successfully removed all references to VSTS?
2. Do the other changes look sensible?
3. Is there anything else you think needs changing in chapter 1? _If it's small I will address here, if it's larger we should open a new item for it._

### CODE REVIEW

File: 01-intro.Rmd [hunk 1]
```diff
@@ -66,14 +66,12 @@ You can think of these sort of systems as wrappers for Git, where merges into th
 
 One of the most well known systems for this is [GitHub](https://github.com) which is popular with open source projects. 
 
-For the purposes of this book we are looking at using Visual Studio Team Services. This is a common tool in enterprise for software development and the reason we use it over GitHub is that it is secure. 
+For the purposes of this book we are looking at using Azure DevOps. This is a common tool in enterprise for software development and the reason we use it over GitHub is that it is secure.
```
Comment (cjrace) [inline 01-intro.Rmd:69]: I'd tweak the wording a little to something like '...over GitHub is that we're able to control access to repositories using our @education.gov.uk accounts.'. It's not that GitHub isn't secure/can't be trusted, it's more that Azure DevOps integrates with our Microsoft accounts (windows login / AAD) and we already have the subscriptions paid for that mean we can authenticate using these. We could do similar in GitHub but we'd have to pay for that, and it wouldn't link with our work accounts.

**Review by cjrace — CHANGES_REQUESTED** [review]
Changes look good - thanks Rachel for doing these!

1. All references to VSTS have been removed.
2. All changes appear sensible.
3. One minor tweak to the original wording has been suggested via a comment.

File: 01-intro.Rmd (hunk 1 above)
Comment (rtadd-git) [reply 01-intro.Rmd]: Thanks - I did have some reservations about this wording too so good to know it wasn't just me. I've had a go at rewording, being mindful that this is effectively a public document and only saying things I'm comfortable with on this basis.

Comment (cjrace) [reply 01-intro.Rmd]: Wording looks good though I'd only recommend DevOps for private repositories. GitHub is a lot nicer to use, and better for public sharing so I'd recommend that for anything that can be public.

Comment (rtadd-git) [reply 01-intro.Rmd]: Good distinction, and not one I'd thought of because my work needs to be private. Now added.

**Review by cjrace — APPROVED** [review]

**Pull request merged**